MetaMorpheus/TaskLayer/SearchTask/PostSearchAnalysisTask.cs

```diff
@@ -199,10 +200,15 @@ private void QuantificationAnalysis()
         {
             if (Parameters.SearchParameters.DoMultiplexQuantification)
             {
-                List<Modification> multiplexMods = Parameters.FixedModifications.Where(m => m.ModificationType == "Multiplex Label").ToList();
-                if (multiplexMods.IsNotNullOrEmpty())
+                List<Modification> possibleMultiplexMods = Parameters.FixedModifications.Where(m => m.ModificationType == "Multiplex Label").ToList();
+                if (possibleMultiplexMods.IsNotNullOrEmpty())
                 {
-                    Parameters.MultiplexModification = multiplexMods.MaxBy(m => m.DiagnosticIons.Count);
+                    MultiplexMod = new(label: possibleMultiplexMods.MaxBy(m => m.DiagnosticIons.Count));
+                }
+
+                foreach(var psm in Parameters.AllPsms)
+                {
+                    psm.SetMultiplexIonIntensities(MultiplexMod);
                 }
                 return;
```

Comment:
For each psm, are you only setting the multiplex (diagnostic) ion intensities of a single modification?

Reply:
Yessir. When Multiplex Quant is enabled, the user can only select one type of multiplex modification (e.g., TMT11 on K + TMT11 on N-terminus). We only need to store the ion intensities for that one multiplex mod.